packages/dashboard/src/pages/routes.tsx

```diff
@@ -16,16 +15,15 @@ const Wallets = lazy(() => import('./Wallets'))
 const Welcome = lazy(() => import('./Welcome'))
 const Personas = lazy(() => import('./Personas'))
 const Settings = lazy(() => import('./Settings'))
+
 export function Pages() {
     return (
         <Suspense fallback="loading...">
             <Switch>
                 <Route path={Routes.Welcome} children={<Welcome />} />
-                <DashboardFrame>
```

Comment:
If it is arranged this way, the state of `DashboardFrame` will be lost in the transition of different routes. This is why I choose to put it here. Can you resolve this problem in your design?

Reply:
I also noticed this problem and the whole root dom tree changed every time I switched routes.I will make some attempts, If I can't resolve, I will add small screen adaptation on your DashboardFrame